### DISCUSSION TITLE
v1.5.1

### DISCUSSION THREAD

**Discussion by Zeal-L** [Zeal-L/BiliBili-Manga-Downloader#129] [Announcements]
![GitHub release (by tag)](https://img.shields.io/github/downloads/Zeal-L/BiliBili-Manga-Downloader/v1.5.1/total)

- **新增功能:**
  - 添加 mac 环境下 的窗口检测（多开）
  - 在UI中添加exif的设置选项，用户可以自行选择是否要往图片属性中插入元信息 (#112)
- **修复bug:**
  - 修复 B漫跟换链接授权规则导致的 Biliplus 无法正常解析 (#125)
  - 修复章节名称带方括号识别库存异常
  - 修复B站二维码登入请求412异常
  - 修正章节标题的错误替换
- **优化设置:**
  - 针对 Macos 更新打包配置
  - 完善对MacOS的支持
  - cbz格式多作者识别优化
  - Linter 由原来的 black 改为 ruff

# 特别感谢
- @dataacat 针对 Biliplus 新规则进行更新适配，为黑科技下载赋能
- @spr-equinox 对 xml 保存格式的更新修复

## What's Changed
* ⬆️ Bump jinja2 from 3.1.2 to 3.1.3 by @dependabot in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/111
* ⬆️ Bump pycryptodomex from 3.19.0 to 3.19.1 by @dependabot in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/116
* 🐛尽力修正标题的错误替换 by @spr-equinox in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/122
* 🐛 修复一个运行时错误 by @spr-equinox in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/123
* 修复BiliPlus Cookie by @dataacat in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/126
* Develop by @Zeal-L in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/127
* 🚧 更新打包配置 by @Zeal-L in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/128

## New Contributors
* @spr-equinox made their first contribution in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/122
* @dataacat made their first contribution in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/126

**Full Changelog**: https://github.com/Zeal-L/BiliBili-Manga-Downloader/compare/v1.5.0...v1.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Zeal-L/BiliBili-Manga-Downloader/releases/tag/v1.5.1'>v1.5.1</a>.</em>

**Comment by iiihazelssl** [comment]
你好，下载1.5.1之后，显示这个是为什么，要怎么解决？
<img width="223" alt="屏幕截图 2024-04-05 134451" src="https://github.com/Zeal-L/BiliBili-Manga-Downloader/assets/165463841/af90e5fb-9bbd-4eed-bd0f-e4444c22510d">

**Reply by iiihazelssl** [reply]
coockie是刚刚复制的，重新登陆过

**Reply by Zeal-L** [reply]
@iiihazelssl 请确认你复制的是整个 cookie，如果还不行，加群解决
![57b14b19be6b252daff7e4d6dbab14f9](https://github.com/Zeal-L/BiliBili-Manga-Downloader/assets/72005386/98b8def3-0b58-4c5c-831d-7b9778e6f309)

**Reply by iiihazelssl** [reply]
你好，按照你的指示图问题已经解决。之前的主要问题是我复制的cookie是access_key里的内容。谢谢。但现在用biliplus解析漫画，章节都未解锁，是由于其他用户没有开启漫画缓存吗？

**Reply by Zeal-L** [reply]
由于叔叔更新了链接分享规则，现在分享漫画的人必须保持登入，别人才能共享到。因此之前好多都能解析的漫画现在都没有了
慢慢等待大佬分享吧，没办法

**Reply by iiihazelssl** [reply]
好的，谢谢👍

**Comment by ywanggmu** [comment]
大佬请问“哔哩哔哩漫画下载器” is damaged and can’t be opened. You should move it to the Trash.如何解决

**Reply by Zeal-L** [reply]
请问是什么系统呢，试过重新下载吗

**Reply by ywanggmu** [reply]
macos Sonoma 14.4.1 m1 试过重新下载了还是不行

**Reply by Zeal-L** [reply]
应该是没有签名导致的安全性策略问题
试试在相应目录打开终端运行下面的命令
`sudo xattr -d com.apple.quarantine 哔哩哔哩漫画下载器.app`

**Comment by kddjdnfn** [comment]
请问bilibiliplus需要付费吗，登陆后有盗号风险吗

**Comment by Archangel415** [comment]
请问双击程序后没有任何反应是怎么回事？后台也没有运行。我是win10系统，下的版本是windows_amd64。

**Comment by TOKEI777** [comment]
你好，请问点击打开应用程序之后出现这个提示是什么意思呀？
![屏幕截图 2025-02-01 132149](https://github.com/user-attachments/assets/cf4fce3d-e556-499e-bb83-6d97cb3e910c)

**Comment by jin00000000000** [comment]
你好，请问大佬为什么我的解析了半个小时没有动静

**Reply by wzymg3** [reply]
叔叔改规则了，已经废了，我去年还被封了一个月

**Comment by kddjdnfn** [comment]
我不是开发者

---- Replied Message ----
| From | ***@***.***> |
| Date | 09/08/2025 下午12:38 |
| To | Zeal-L/BiliBili-Manga-Downloader ***@***.***> |
| Cc | kddjdnfn ***@***.***>,
Comment ***@***.***> |
| Subject | Re: [Zeal-L/BiliBili-Manga-Downloader] v1.5.1 (Discussion #129) |

叔叔改规则了，已经废了，我去年还被封了一个月

**Comment by Connor888t** [comment]
你好！你群的115资源解压密码多少？拜托
